tests/test_class2D.py

```diff
@@ -276,7 +307,10 @@ def testImplementations(self):
         # Nearest Neighbhor component
         with pytest.raises(ValueError, match=r"Provided nn_implementation.*"):
             _ = RIRClass2D(
-                self.clean_src, self.clean_fspca_basis, nn_implementation="badinput"
+                self.clean_src,
+                self.clean_fspca_basis,
+                bispectrum_components=int(0.75 * self.clean_fspca_basis.basis.count),
```

Comment:
Is this line just to avoid the `fspca_components < bispectrum_components` error?